ci/workflows/main-cron.yml

```diff
@@ -302,3 +302,16 @@ steps:
           config: ci/docker-compose.yml
           mount-buildkite-agent: true
     timeout_in_minutes: 20
+
+  # Sqlsmith differential testing
```

Comment:
Hmm do we rly need a separate workflow?

What’s wrong with merging into the existing sqlsmith but additionally run differential testing on each of the created MVs?

Reply:
Anw, not a big concern, just wanted to hear your reasoning. I can see why separate workflow can be preferred as it separates panic discovery and logical errors.

Reply:
> I can see why separate workflow can be preferred as it separates panic discovery and logical errors.

Yeah that's the rough idea. It's mainly to decouple 2 different testing workflows. Makes it easier to debug any issues and maintain.